src/pages/Homepage.jsx

```diff
@@ -75,7 +88,7 @@ const Homepage = () => {
     >
       <TimerComponent />
       <ActiveTeamsComponent
-        teams={teams}
+        teams={teams.filter((team) => team.active)}
```

Comment:
Here I only pass teams into the component which are relevant to minimize memory impact.

Reply:
It also automatically handles the div that'll show when the ul element is empty. Noted.